staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.proto

```diff
@@ -43,6 +43,13 @@ service DRAPlugin {
   // The same error handling rules apply,
   rpc NodeUnprepareResources (NodeUnprepareResourcesRequest)
     returns (NodeUnprepareResourcesResponse) {}
+
+  // CheckDeviceAllocation prevent from that same device to be assigned to different claims.
+  // DRA plugin authors should check for the fellow situation:
+  // i. allocation by different schedulers without coordination
+  // ii. device state change
+  rpc CheckDeviceAllocation (CheckDeviceAllocationRequest)
```

Comment:
API in this file must be implemented by the plugin and called by Kubelet. According to what @pohly said, this is not the way we want it to be, as far as I understand.